tasks/check-settings-centos.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: "Check for innodb_log_file_size setting (RedHat/CentOS)"
+  shell:
+    cmd: grep -E ^innodb_log_file_size /etc/my.cnf | awk -F= '{ print $2}' | sed 's/\s//g'
+    removes: "/etc/my.cnf"
+  register: configured_innodb_log_file_size
+
+- name: "Abort when innodb_log_file_size changes"
```

Comment:
Hi @mamedin I'm not clear why this step is here - is it required for MySQL <=5.6.7 because extra steps are necessary to change the `innodb_log_file_size` and the changes should be done manually?

Reply:
Yes, for 5.7 it could be changed directly, but <=5.6.7 it needs a procedure.

How to increase innodb_log_file_size on running system:

Set innodb_fast_shutdown=1:

```
sudo mysql -e "SET GLOBAL innodb_fast_shutdown = 1;"
```

Set innodb_log_file_size = 1G  on /etc/mysql/my.cnf file:

```
innodb_log_file_size = 1G 
``` 
Move or delete the innodb files at /var/lib/mysql directory:
```
cd /var/lib/mysql
sudo mv ib_logfile0 ib_logfile0.bak
sudo mv ib_logfile1 ib_logfile1.bak
```
Restart mysql:
```
sudo service mysql restart
``` 
Delete old innodb files if mysql works fine:
```
sudo rm ib_logfile*.bak 
```